modules/nw-operator-cr.adoc

```diff
@@ -289,6 +289,20 @@ endif::ibm-cloud[]
 ====
  While migrating egress traffic, you can expect some disruption to workloads and service traffic until the Cluster Network Operator (CNO) successfully rolls out the changes.
 ====
+
+|`v4InternalSubnet`
+|
+If your existing network infrastructure overlaps with the `100.64.0.0/16` IPv4 subnet, you can specify a different IP address range for internal use by OVN-Kubernetes. You must ensure that the IP address range does not overlap with any other subnet used by your {product-title} installation. The IP address range must be large enough to allocate one IP address per cluster node.
+
+This field cannot be changed after installation.
+|The default value is `100.64.0.0/16`.
+
+|`v6InternalSubnet`
+|
+If your existing network infrastructure overlaps with the `fd98::/48` IPv6 subnet, you can specify a different IP address range for internal use by OVN-Kubernetes. You must ensure that the IP address range does not overlap with any other subnet used by your {product-title} installation. The IP address range must be large enough to allocate one IP address per cluster node.
```

Comment:
ditto

Reply:
@pliurh, I don't understand enough about IPv6 to be able to translate. It's probably not a `/24` though?

Reply:
The calculation is similar to the IPv4. If the clusterNetwork CIDR is fd01::/56, and the hostPrefix is /64. The maximum number of nodes is 2^(64-56)=256. And as we need one IP for the gateway, one for the network, and one for broadcasting. So the internal IP range must be at least /23.